twill-common/src/main/java/org/apache/twill/filesystem/LocalLocation.java

```diff
@@ -251,6 +251,34 @@ public boolean mkdirs() throws IOException {
     return file.mkdirs();
   }
 
+  @Override
+  public boolean mkdirs(String permission) throws IOException {
```

Comment:
Should we use  [Files.createDirectories](http://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createDirectories(java.nio.file.Path,%20java.nio.file.attribute.FileAttribute...)) since it also takes permissions?